src/view/StrokeView.cpp

```diff
@@ -42,25 +42,29 @@ void StrokeView::changeCairoSource(bool markAudioStroke) {
     ///////////////////////////////////////////////////////
     // Fill stroke
     ///////////////////////////////////////////////////////
+    bool audioStrokebutNoAudio = markAudioStroke && s->getAudioFilename().length() == 0;
```

Comment:
This variable's name is a bit confusing, I think. Maybe a simpler `fadeOutStroke` would be clearer?